### PR TITLE
documentation(EJ2-59479): add doc for binding data on foreign key col…

### PR DESCRIPTION
**Pull request by anbuarjun**
…umns.

### CODE REVIEW

**Comment by Lakshmipriya07** [comment]
Content is fine.

**Comment by BalajiSGrid** [comment]
Changes are fine

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ In the following example, The **Employee Name** is a foreign key column and whil
 
 ## Customize filter UI in foreignkey column
 
-You can create your own filtering UI by using [`filter`](https://helpsyncfusion.com/cr/cref_files/aspnetcore-js2/aspnetcore/Syncfusion.EJ2~Syncfusion.EJ2.Grids.GridColumn~Filter.html) property of [`e-grid-column`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html) tag helper. The following example demonstrates the way to create a custom filtering UI in the foreign column.
+You can create your own filtering UI by using [`filter`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html#Syncfusion_EJ2_Grids_GridColumn_Filter) property of [`e-grid-column`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html) tag helper. The following example demonstrates the way to create a custom filtering UI in the foreign column.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md:84]: remove tilde(` `)symbol from API

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md [hunk 2]
```diff
@@ -169,3 +169,33 @@ In the following example, The **Employee Name** is a foreign key column and the
 {% endtabs %}
 {% endif %}
 
+
+## Bind data on foreign key columns with UrlAdaptor
+
+You can bind the data on foreign key columns using `UrlAdaptor`. In the following example, **EmployeeID** is designated as a [`ForeignKeyField`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html#Syncfusion_EJ2_Grids_GridColumn_ForeignKeyField), and **ShipName** is designated as a [`ForeignKeyValue`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html#Syncfusion_EJ2_Grids_GridColumn_ForeignKeyValue).
+
+To communicate with a specific DataSource, DataManager uses the adaptor idea. UrlAdaptor is used to interact with remote data services and to bind data on foreign key columns.
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md:177]: first line is not suits in this place.

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md [hunk 3]
```diff
@@ -169,3 +169,33 @@ In the following example, The **Employee Name** is a foreign key column and the
...
+
+<!-- We attach data from datasource UserEditingSpeciesSiteLinksQuery to a foreign key column with using url adaptor. -->
```
Comment (MaithiliyK) [inline ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md:179]: Here "UserEditingSpeciesSiteLinksQuery" what it means and maintain consistency while adaptor names

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md (hunk 1 above)
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md]: I removed that mam.

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md [hunk 4]
```diff
@@ -169,3 +169,33 @@ In the following example, The **Employee Name** is a foreign key column and the
 {% endtabs %}
 {% endif %}
 
+
+## Bind data on foreign key columns with UrlAdaptor
+
+You can bind the data on foreign key columns using `UrlAdaptor`. In the following example, **EmployeeID** is designated as a [`ForeignKeyField`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html#Syncfusion_EJ2_Grids_GridColumn_ForeignKeyField), and **ShipName** is designated as a [`ForeignKeyValue`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Grids.GridColumn.html#Syncfusion_EJ2_Grids_GridColumn_ForeignKeyValue).
```
[the remark this answers is not in the document]
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md]: I removed and corrected that changes mam.

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md (hunk 2 above)
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md]: I modified that line mam.

File: ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md (hunk 3 above)
Comment (anbuarjun) [reply ej2-asp-core-mvc/grid/EJ2_ASP.NETCORE/columns/foreign-key-column.md]: no need this line on that place mam, so I commented that line mam. Now I removed that line mam.

**Comment by BalajiSGrid** [comment]
Changes are fine so, please merge it

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**